packages/devtools_extensions/example/README.md

```diff
@@ -1,46 +1,126 @@
 # Example DevTools Extension
 
-This is an end-to-end example of a DevTools extension, including
-- the parent `package:foo` that provides the DevTools extension to end-user
-applications (`foo/packages/foo`)
-- the end-user application (`app_that_uses_foo`) that depends on `package:foo`,
-which will trigger a load of the `package:foo` DevTools extension when debugging
-this app with DevTools
-- the `package:foo` DevTools extension (`foo/packages/foo_devtools_extension`),
-which is a Flutter web app that will be embedded in DevTools when debugging an
-app the uses `package:foo`
-
-
-This example will show you how to:
-1. Structure your package for optimal extension development and publishing
+This directory contains end-to-end examples of DevTools extensions. Each
+end-to-end example is made up of three components:
+
```

Comment:
It will be easier to read if you could introduce simple short name for each component, itemize them in short  here and then use these names when explaining or referencing them.

Reply:
done.